drivers/si70xx/si70xx.c

```diff
@@ -44,47 +50,70 @@ int si70xx_test(const si70xx_t *dev)
     uint8_t revision = si70xx_get_revision(dev);
 
     if (revision != SI70XX_REVISION_1 && revision != SI70XX_REVISION_2) {
-        return -1;
+        DEBUG("[ERROR] Bad device revision (%d).\n", revision);
+        return -SI70XX_ERR_NODEV;
     }
 
     uint8_t id = si70xx_get_id(dev);
 
-    if (id != SI70XX_ID_SI7006 && id != SI70XX_ID_SI7013 &&
-        id != SI70XX_ID_SI7020 && id != SI70XX_ID_SI7021) {
-        return -2;
+#if defined(MODULE_SI70XX_ID_SI7006)
```

Comment:
The pseudomodules are called just e.g., "si7006", right?

Anyhow, please move the definition of "SI70XX_ID" (and possibly of the name) to file-level.